content/client-server-api/modules/report_content.md

```diff
@@ -18,3 +18,6 @@ Servers are free to handle the reported content however they desire.
 This may be a dedicated room to alert server administrators to the
 reported content or some other mechanism for notifying the appropriate
 people.
+
+The server MUST verify that the user has permission to view the event
+before accepting a report.
```

Comment:
Again, this feels looser than what the MSC stated, which is that the reporter must currently be joined to the room that the reported event is in. Perhaps:

```suggestion
The server MUST verify that the user reporting the event is currently
joined to the room the event is in before accepting a report.

Reply:
further, a `changed-in` annotation:


```suggestion
{{< changed-in v="1.7" >}} The server MUST verify that the user 
reporting the event is currently joined to the room the event is 
in before accepting a report.
```